bin/merge_sces.R

```diff
@@ -63,11 +63,14 @@ if(is.null(opt$input_library_ids)){
   # extract library ids from filename
   input_library_ids <- stringr::word(basename(input_sce_files), 1, sep = "_")
 } else {
-  # pull library ids from list 
-  input_library_ids <- unlist(stringr::str_split(opt$input_library_ids, ',')) 
+  # pull library ids from list
+  input_library_ids <- unlist(stringr::str_split(opt$input_library_ids, ','))
```

Comment:
We could save a step here, since we don't use the library_ids any other way (right?)
```suggestion
  names(input_sce_files) <- unlist(stringr::str_split(opt$input_library_ids, ','))
```